src/chttpd/src/chttpd.erl

```diff
@@ -677,7 +677,7 @@ absolute_uri(#httpd{absolute_uri = URI}, Path) ->
     URI ++ Path.
 
 unquote(UrlEncodedString) ->
-    mochiweb_util:unquote(UrlEncodedString).
+    chttpd_util:unquote(UrlEncodedString).
```

Comment:
We shouldn't change `chttpd:unquote/1`. Unquote is used everywhere and should keep converting + into space. 

The only exception when it shouldn't is for path component of the URL. I suggest to create `chttpd:unquote_path/1`

Reply:
Fixed in https://github.com/apache/couchdb/pull/3638/commits/10061e63a196c0851cd0894849034ee501858111.